src/app/components/panel/__snapshots__/Panel.test.js.snap

```diff
@@ -0,0 +1,278 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Note this is a generated file which is used in the tests not too much reviewing to be done here.